platform-tests/functional-tests/src/test/java/org/ekstep/platform/content/ContentAcceptFlagV3Test.java

```diff
@@ -0,0 +1,378 @@
+            then().
+                body("responseCode", equalTo("OK"));
+    }
+
+    private void flag(String contentId) {
+        String versionKey = getVersionKey(contentId);
+        String flagRequestBody = "{\"request\": {\"flagReasons\":[\"Copyright Violation\"],\"flaggedBy\":\"gauraw\",\"versionKey\": \"" + versionKey + "\"}}";
+        given().
+                spec(getRequestSpecification(contentType, userId, APIToken)).
+                body(flagRequestBody).
+                with().
+                contentType(JSON).
+                when().
+                post("/content/v3/flag/" + contentId).
+                then().
+                body("responseCode", equalTo("OK"));
+    }
+
+    private void acceptFlag(String contentId) {
+        String versionKey = getVersionKey(contentId);
+        String acceptFlagRequestBody = "{\"request\": {\"versionKey\":\""+ versionKey +"\"}}";
+        given().
+                spec(getRequestSpecification(contentType, userId, APIToken)).
+                body(acceptFlagRequestBody).
+            with().
+                contentType(JSON).
+            when().
+                post(BASE_PATH + "/flag/accept/" + contentId).
+            then().
+                body("responseCode", equalTo("OK"));
+    }
+
+    private Response acceptFlagAndGetResponse(String contentId) {
+        String versionKey = getVersionKey(contentId);
+        String acceptFlagRequestBody = "{\"request\": {\"versionKey\":\""+ versionKey +"\"}}";
+        return given().
+                spec(getRequestSpecification(contentType, userId, APIToken)).
+                    body(acceptFlagRequestBody).
+                with().
+                    contentType(JSON).
+                when().
+                    post(BASE_PATH + "/flag/accept/" + contentId).
+                then().
+                    extract().response();
+    }
+
+    public Response getContentResponse(String contentId) {
+        return given().
+                    spec(getRequestSpecification(contentType, validuserId, APIToken, channelId, appId)).
+                with().
+                    contentType(JSON).
+                when().
+                    get(BASE_PATH + "/read/" + contentId).
+                then().
+                    body("responseCode", equalTo("OK")).
+                extract().
+                response();
+    }
+
+    @Test
+    public void acceptFlaggedDocumentContent() {
+        setURI();
+        String contentId = createDocumentContent();
+        upload(contentId);
+        publish(contentId);
+        delay(25000);
+        flag(contentId);
+        acceptFlag(contentId);
+        Response imageNodeResponse = getContentResponse(contentId + DEFAULT_CONTENT_IMAGE_OBJECT_SUFFIX);
+        assertEquals("FlagDraft", imageNodeResponse.jsonPath().get("result.content.status"));
+        Response response = getContentResponse(contentId);
+        assertEquals("Retired", response.jsonPath().get("result.content.status"));
```

Comment:
assert statement is missing for Response Code.

Reply:
Validation for response code is written in getContentResponse(String) method.